website/docs/r/branch_protection.html.markdown

```diff
@@ -49,7 +48,11 @@ resource "github_branch_protection" "example" {
 
 }
 
-resource "github_user" "example" {
+resource "github_repository" "example" {
+  name = "test"
+}
+
+data "github_user" "example" {
```

Comment:
This was a `github_user` resource, but should have been a data source as far as I can tell.